### ISSUE TITLE
unable to GKE update node version when there are multiple pools

### ISSUE THREAD

**Issue by Stono** [terraform-providers/terraform-provider-google#1274]
Hi there,
We have quite a complex GKE setup which uses a `google_container_cluster`, and then 3x `google_container_node_pool` as well.

When we come to update the node_version today, we're getting this error:
```
* module.k8-elastic-cluster.google_container_cluster.cluster: 1 error(s) occurred:

* google_container_cluster.cluster: googleapi: Error 400: Node pool id must be specified if cluster has more than one node pool.., badRequest
```

So it looks like the google api is wanting you to specify which pool to upgrade so behind the scenes you'd need to iterate over each `node_pool` and one by one (as you can only do one at a time) upgrade them.

**Future**
I would suggest that the `node_version` should actually be an attribute on `google_container_node_pool`, as pools can be on different versions.

**Comment by Stono** [comment]
Linking https://github.com/terraform-providers/terraform-provider-google/pull/1266

**Comment by danawillow** [comment]
Closing this since it's a duplicate of #633 and is already fixed by #1266 (which will be in the next release, likely next week but no promises)
